homeassistant/components/xiaomi_aqara/__init__.py

```diff
@@ -351,9 +352,13 @@ def _async_track_unavailable(self):
             return True
         return False
 
+    def push_data(self, data: dict[str, Any], raw_data: dict[Any, Any]) -> None:
+        """Push from Hub running in another thread."""
+        self.hass.loop.call_soon(self.async_push_data, data, raw_data)
```

Comment:
Ideally we would have a `hass.add_hass_job` and wrap this in `HassJob` but the call_soon is probably fine for now

Reply:
This doesn’t actually fix the problem because it should have been call_soon_threadsafe

Reply:
Will do a follow up tomorrow.